web-ui/router.ml

```diff
@@ -13,6 +13,7 @@ let create ~github ~gitlab =
       Dream.get "/css/ansi.css" (fun _ ->
           Dream.respond ~headers:[ ("content-type", "text/css") ] Ansi.css);
       Dream.get "/css/**" @@ Dream.static ~loader "/css";
+      Dream.get "/images/**" @@ Dream.static ~loader "/images";
```

Comment:
This is more a side note for us than a request, but we might need to separate static files in a `static` sub dir we can mount with `ocaml-crunch` at some point.